### ISSUE TITLE
tlimit('full') not working

### ISSUE THREAD

**Issue by jameswilburlewis** [MAVENSDC/PyTplot#181]
In the following code fragment:

```
        vars = fgm(probe='a',level='l2',trange=['2007-03-23', '2007-03-24'])
        tplot('tha_fgl_dsl')
        tlimit(['2007-03-23/14:00','2007-03-23/14:00:00.9'])
        tplot('tha_fgl_dsl')
        tlimit('full')
        tplot('tha_fgl_dsl')

```

the tlimit('full') call does not change the displayed time range.

**Comment by ericthewizard** [comment]
Try tlimit(full=True)

**Comment by jameswilburlewis** [comment]
That works, thanks!  I guess the full==True and arg=='full' cases should be merged, same for 'last'.

**Comment by jameswilburlewis** [comment]
Fixed in pytplot-mpl-temp 2.1.41
